devel/ninja/Portfile

```diff
@@ -3,6 +3,11 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
+platform darwin 8 {
+    PortGroup       legacysupport 1.1
+    legacysupport.newest_darwin_requires_legacy 10
```

Comment:
This should be done differently.
```
PortSystem          1.0
PortGroup           github 1.0
PortGroup           legacysupport 1.1

legacysupport.newest_darwin_requires_legacy 8
```
(Assuming it is needed for Tiger.)

Could you share a log what actually fails with the current Ninja? (Sorry, if I forgot, but in any case it is rather be referenced in the PR, so that the reason for a fallback is clear.)